featurebyte/routes/event_table/api.py

```diff
@@ -145,7 +146,7 @@ async def get_event_table_info(
         """
         controller = self.get_controller_for_request(request)
         info = await controller.get_info(
-            document_id=event_table_id,
+            document_id=ObjectId(event_table_id),
```

Comment:
Will we get type check or lint error if we forgot to apply this conversion here?

Reply:
We will get type check error here though `event_table_id` is already an `ObjectId`. I think the reason is that `PyObjectId` is inherited `str` type.